src/MIDebugEngine/MIDebugEngine.csproj

```diff
@@ -70,6 +70,7 @@
   <ItemGroup Label="NuGet Packages">
     <PackageReference Include="Microsoft.VisualStudio.Debugger.Interop.Portable" Version="$(Microsoft_VisualStudio_Debugger_Interop_Portable_Version)" />
     <PackageReference Include="Microsoft.VisualStudio.Debugger.Interop.15.0" Version="$(Microsoft_VisualStudio_Debugger_Interop_15_0_Version)" />
+    <PackageReference Include="Microsoft.VisualStudio.Debugger.Interop.16.0" Version="$(Microsoft_VisualStudio_Debugger_Interop_16_0_Version)" />
```

Comment:
You will want to add this reference in [MIDebugPackage.csproj](https://github.com/microsoft/MIEngine/blob/74f810009496b87a0f6a7d592d691a303e099656/src/MIDebugPackage/MIDebugPackage.csproj#L78) and [OpenDebugAD7.csproj](https://github.com/microsoft/MIEngine/blob/74f810009496b87a0f6a7d592d691a303e099656/src/OpenDebugAD7/OpenDebugAD7.csproj#L110).

So it is packaged for MDD for VS and XPlat for VS Code.